src/models/Movie/parseMovieData.ts

```diff
@@ -1,15 +1,38 @@
-import { MovieSummary } from './type';
+import { Movie, MovieSummary } from './type';
+import { GenreContentfulEntry } from '../Genre/GenreModel';
 
 // TODO: Fix contentful types, take out the any
-export default function parseMovieData(entry: any): MovieSummary {
+
+// Function to parse basic movie data (MovieSummary)
+export function parseMovieSummary(entry: any): MovieSummary {
 	const { sys, fields } = entry;
 
-	const movie: MovieSummary = {
+	return {
 		id: sys.id,
 		title: fields.title || '',
 		posterUrl: fields.posterUrl || '',
 		rating: fields.rating || '',
 	};
+}
 
-	return movie;
+// Function to parse detailed movie data (MovieModel)
+export function parseMovieModel(entry: any): Movie {
```

Comment:
nit: this could be just `parseMovie` since it's not parsing to a model (just a simple type.